json_logic/__init__.py

```diff
@@ -100,6 +100,9 @@ def merge(*args):
 def in_operator(a, b):
     """Implements the 'in' operator for work with lists."""
     if "__contains__" in dir(b):
+        if isinstance(a, str):
+            # case-insensitive checking string in string list
+            return a.casefold() in [string.casefold() for string in b if isinstance(string, str)]
```

Comment:
What if the list has non-strings? Please cover with tests properly considering various cases.

Reply:
It is checked in `if` condition in the end of lambda expression